### PR TITLE
[GulpV0] Migrated to Node10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: GulpV0

**Description**: Migrated GulpV0 to Node 10 execution handler.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** No

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by egor-bryzgalov — COMMENTED** [review]
I think we should cover the task with a [Canary test](https://github.com/microsoft/build-task-team/issues/387) first.

**Review by anatolybolshakov — APPROVED** [review]
LGTM, but let's cover this task by e2e tests first.

**Review by EzzhevNikita — APPROVED** [review]

**Comment by DaniilShmelev** [comment]
This task has legacy tests, would you mind migrating them?

File: Tasks/GulpV0/task.json
Comment (DaniilShmelev) [inline Tasks/GulpV0/task.json:19]: Also: this task seems to be due for [sprint 182](https://github.com/microsoft/build-task-team/milestone/7); could you please fix this in task.json and task.loc.json?

Comment (anatolybolshakov) [reply Tasks/GulpV0/task.json:19]: Bumped task version

**Comment by anatolybolshakov** [comment]
Let's merge this after merging of related canary PR

**Review by DaniilShmelev — APPROVED** [review]

**Comment by sdobrodeev** [comment]
> This task has legacy tests, would you mind migrating them?

Created https://github.com/microsoft/build-task-team/issues/716 for it

**Pull request merged**
